modules/swagger-codegen/src/main/java/io/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -1,39 +1,17 @@
 package io.swagger.codegen.examples;
 
-import static io.swagger.models.properties.StringProperty.Format.URI;
-import static io.swagger.models.properties.StringProperty.Format.URL;
-
 import io.swagger.models.Model;
 import io.swagger.models.ModelImpl;
-import io.swagger.models.properties.ArrayProperty;
-import io.swagger.models.properties.BaseIntegerProperty;
-import io.swagger.models.properties.BooleanProperty;
-import io.swagger.models.properties.DateProperty;
-import io.swagger.models.properties.DateTimeProperty;
-import io.swagger.models.properties.DecimalProperty;
-import io.swagger.models.properties.DoubleProperty;
-import io.swagger.models.properties.FileProperty;
-import io.swagger.models.properties.FloatProperty;
-import io.swagger.models.properties.LongProperty;
-import io.swagger.models.properties.MapProperty;
-import io.swagger.models.properties.ObjectProperty;
-import io.swagger.models.properties.Property;
-import io.swagger.models.properties.RefProperty;
-import io.swagger.models.properties.StringProperty;
-import io.swagger.models.properties.UUIDProperty;
+import io.swagger.models.properties.*;
```

Comment:
Please keep the explicit import instead of using a wildcard as suggested by the Google Java style guide: https://google.github.io/styleguide/javaguide.html